include/aspect/initial_conditions/S40RTS_perturbation.h

```diff
@@ -31,7 +31,7 @@ namespace aspect
   {
     using namespace dealii;
 
-    namespace internal
+    namespace S40RTSinternal
```

Comment:
Just make sure. So I'll adjust the code to aspect::InitialConditions::internal::S40RTS?